modules/backup.js

```diff
@@ -1,33 +1,49 @@
 /**
  * @access public
- * @description Saves specified file type to external storage
- *   device. Callback is invoked upon completion. If an error
- *   occurred, it will be passed as the first parameter to
- *   the callback, "oncomplete".
- * @param {string} type
+ * @description Backups every data type set as true in settings
+ * Calls the callback on every error
  * @param {callback} oncomplete
- * @throws On invalid data type
  */
-var backup = function(type, oncomplete) {
+var backup = function(oncomplete) {
```

Comment:
We may want to consider having a callback function that gets called after each type updates for progress reports